src/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "awesome-emoji-picker",
+  "version": "2.0.0",
```

Maybe this should be 3.0 as well:
```suggestion
  "version": "3.0.0",
```